libgdf/src/tests/column/column-test.cu

```diff
@@ -279,3 +280,28 @@ TEST_F(ColumnConcatTest, Benchmark) {
                         bit_setter);
 }
 #endif // ENABLE_CONCAT_BENCHMARK
+
+
+TEST(ColumnByteWidth, TestByteWidth)
```

Comment:
Is this really the only test of the type dispatcher? Is this sufficient?

Reply:
All of the existing join/groupby/hash tests will exercise the `gdf_table` functions that are updated to use `gdf_type_dispatcher`.